cache.rst

```diff
@@ -714,3 +714,119 @@ Clear all caches everywhere:
 .. code-block:: terminal
 
     $ php bin/console cache:pool:clear cache.global_clearer
+
+Encrypting the Cache
+--------------------
+
```

Comment:
We always add a little box that indicates that this is a new feature:

```rst
.. versionadded:: 5.2

    The ``SodiumMarshaller`` class was introduced in Symfony 5.2.
```

Reply:
Added (for 5.1).